plugins/ltac2/tac2print.ml

```diff
@@ -189,7 +189,9 @@ let pr_factorized_constructor pr_rec lvl tpe = function
       let args = List.combine def cl in
       let pr_arg ((id, _, _), arg) =
         let kn = change_kn_label tpe id in
-        pr_projection kn ++ spc () ++ str ":=" ++ spc () ++ pr_rec E1 arg
+        pr_projection kn ++
```

Comment:
I'm not sure it's a great idea to systematically apply punning when printing, if anything this is somewhat confusing. I believe on the contrary that Ltac2 printing should remove as much sugar as possible so that it becomes clear what is sent to the interpreter.

Reply:
ok